plugins/reach/plugins/OpenCalais/batch/kReachVendorTaskOpenCalaisProcessorEngine.php

```diff
@@ -551,7 +551,29 @@ protected function getTargetMetadataFields (KalturaMetadataProfile $metadataProf
                 $metadataProfileIdXpath = "/xsd:schema/xsd:element/xsd:complexType/xsd:sequence/xsd:element[@name='$fieldName']/xsd:annotation/xsd:appinfo/metadataProfileId";
 
                 $this->targetAllMetadataFields[$fieldName]['metadataProfileId'] = intval($xml->xpath($metadataProfileIdXpath)[0]);
-            }
+
+				$additionalConditionsXpath = "/xsd:schema/xsd:element/xsd:complexType/xsd:sequence/xsd:element[@name='$fieldName']/xsd:annotation/xsd:appinfo/additionalConditions";
+				$additionalConditions = $xml->xpath($additionalConditionsXpath);
+
+				if (count($additionalConditions))
+				{
+					$this->targetAllMetadataFields[$fieldName]['additionalConditions'] = array();
```

Comment:
Using the same const values additionalConditions and fieldName and validValue as strings is wrong, please define as consts.